x-pack/plugins/ingest_pipelines/__jest__/client_integration/ingest_pipelines_create.test.tsx

```diff
@@ -185,30 +185,5 @@ describe('<PipelinesCreate />', () => {
         expect(find('savePipelineError').find('li').length).toBe(8);
       });
     });
-
-    describe('test pipeline', () => {
```

Comment:
Tests for the test pipeline functionality now live in https://github.com/elastic/kibana/blob/master/x-pack/plugins/ingest_pipelines/public/application/components/pipeline_processors_editor/__jest__/test_pipeline.test.tsx. I think it's repetitive to has this test here, and the other test suite goes into much more complexity.